admin_scripts/update_all.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
```

Comment:
This script is missing our template header.

Reply:
Addressed in commit 4e65effe16a7fab99deb91c8943e446ad1a5007f.